front/tests/010-rollingstock-editor.spec.ts

```diff
@@ -0,0 +1,192 @@
+import { test, expect } from '@playwright/test';
+import { RollingStock } from 'common/api/osrdEditoastApi';
+import { PlaywrightRollingstockEditorPage } from './pages/rollingstock-editor-page-model';
+import PlaywrightCommonPage from './pages/common-page-model';
+import { deleteApiRequest } from './assets/utils';
+
+test.describe('Rollingstock editor page', () => {
+  test('should correctly create a new rolling stock (enabled)', async ({ page }) => {
+    const playwrightCommonPage = new PlaywrightCommonPage(page);
+    const playwrightRollingstockEditorPage = new PlaywrightRollingstockEditorPage(page);
+
+    await page.goto('/rolling-stock-editor');
+
+    await playwrightRollingstockEditorPage.clickOnNewRollingstockButton();
+
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('name', 'Test_name');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('detail', 'Test detail');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('family', 'Test family');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('grouping', 'Test grouping');
+
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('number', 500);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('reference', 'Test reference');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('series', 'Test series');
+
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('subseries', 'Test subseries');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('type', 'Test type');
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('unit', 'Test unit');
+
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('length', 50);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('mass-input', 80, true);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('maxSpeed', 200);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('startupTime', 5);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('startupAcceleration', 5);
+
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('comfortAcceleration', 1);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('inertiaCoefficient', 1);
+    await playwrightRollingstockEditorPage.fillAndCheckInputById('gammaValue', 1);
```

Comment:
For all these repetitive lines, couldn't we loop on an array or object with all needed parameters ?
Like : 
```
const rollingStockTestForm = [["name", "Test_name"], ["detail", "Testdetail"]]
or
const rollingStockTestForm = [{
      property: "name", 
      label: "Test_name"
   }, {
      property: "detail", 
      label: "Test detail"
   }
]
```

Reply:
As we've seen, async is not well managed in loops.